staging/src/k8s.io/api/core/v1/annotation_key_constants.go

```diff
@@ -95,6 +95,10 @@ const (
 	// configuration of a resource for use in a three way diff by UpdateApplyAnnotation.
 	LastAppliedConfigAnnotation = kubectlPrefix + "last-applied-configuration"
 
+	// DefaultContainerAnnotationName is an annotation name that can be used to preselect the interesting container
+	// from a pod when running kubectl.
+	DefaultContainerAnnotationName = kubectlPrefix + "default-container"
```

Comment:
I actually don't know if this is really the right place. There's the well_known_labels.go, but well, this is not a label right?

Anyway, there should have a better place to get this (and the documentation be generated automatically).

We can ask later for folks of api-machinery or sig-architecture the best place to put this const :)

Reply:
So I will change it later for the annotation location.

Reply:
Yeah, I think there's no need to put that in the API package. Let's keep it inside kubectl only.